EDGAR/parser.py

```diff
@@ -110,10 +110,26 @@ def _parse_annotated_text(self, driver_path: str, highlight: bool = False, save:
 
         annotation_dict = dict()
         for i in range(len(found)):
+            in_table = False
+            
             found_annotation = found[i].find_elements(By.TAG_NAME, 'ix:nonnumeric')
             found_annotation += found[i].find_elements(By.TAG_NAME, 'ix:nonfraction')
-            annotation_dict[found[i]] = found_annotation
-
+            
+            current_element = found[i]
+            while not in_table: # loop through ancestors
+                # # check if current is root
+                try:
+                    parent = current_element.parent
+                except AttributeError: # if there is no parent => it is the root
+                    break
+                if current_element.tag_name == 'table':
+                    in_table = True
+                current_element = parent
+            
+            if not in_table:
+                annotation_dict[found[i]] = found_annotation
+            else:
+                annotation_dict[found[i]] = []
```

Comment:
You should rebase onto edgar and repush your branch. This looks like outdated code being re-added to the EDGAR branch because you haven't updated your branch in a while.

Reply:
That's weird, I rebased the branch at the beginning of the class. I will redo it to see if it can solve the problem.

Reply:
Did you ```git checkout EDGAR ; git pull ; git checkout nczhao ; git rebase EDGAR```?

Reply:
I might missed the checkout step.